docs: IIIF image support (DEV-3618) (#973)
dasch-swiss/dsp-tools: docs/file-formats/xml-data-file.md

The following property elements exist:

- `<bitstream>`: contains a path to a file (if the resource is a multimedia resource)

> - `<iiif-uri>`: contains a URI to an IIIF image server (resource must be a `StillImageRepresentation`)

Reviewer: the part in brackets should be identical to the line above. You can change either the line above, or this line, but I find it confusing if the same thing is described in 2 different ways

Reviewer: Sorry, my fault. Sure, this is correct, you can leave it as it is 👍

- There is only _one_ `<bitstream>` element allowed per representation.
- The `<bitstream>` element must be the first element.

> - If a `<bitstream>` element is present, the `<iiif-uri>` element must be omitted.

Reviewer: There is no `<iiif-uri>` element in the first place, so it cannot be omitted. 

```suggestion
- If a resource has a `<bitstream>` element, it cannot have a `<iiif-uri>` element.
```

</resource>
```

### `<iiif-uri>`

The `<iiif-uri>` element is used for IIIF image server URIs. 
All versions of the IIIF-API standards are supported.

Notes:

- Only _one_ `<iiif-uri>` element is allowed per `StillImageRepresentation`. 

> - If a `<iiif-uri>` element is present, the `<bitstream>` element must be omitted.

Reviewer: ```suggestion
- If a resource has a `<iiif-uri>` element, it cannot have a `<bitstream>` element.
```
